neo4j-jdbc/src/main/java/org/neo4j/jdbc/ConnectionImpl.java

```diff
@@ -66,12 +71,16 @@
  * @author Michael J. Simons
  * @since 6.0.0
  */
-final class ConnectionImpl implements Neo4jConnection {
+final class ConnectionImpl implements ExtendedNeo4jConnection {
+
+	private static final Logger LOGGER = Logger.getLogger(Neo4jConnection.class.getCanonicalName());
 
 	private static final int TRANSLATION_CACHE_SIZE = 128;
 
 	private final BoltConnection boltConnection;
 
+	private final Set<AutoCloseable> instantiatedAutoClosables = new HashSet<>();
```

Comment:
Naming: I think `trackedStatements` is better.

Reply:
Good point, I was actually going to find a better name, but my focus moved to the timeout implementation.